src/main/java/com/samsolutions/employeesdep/model/converters/EmployeeEntityToDTOConverter.java

```diff
@@ -0,0 +1,30 @@
+package com.samsolutions.employeesdep.model.converters;
+
+import com.samsolutions.employeesdep.model.dto.EmployeeDTO;
+import com.samsolutions.employeesdep.model.dto.RoleDTO;
+import com.samsolutions.employeesdep.model.entities.Employee;
+import com.samsolutions.employeesdep.model.entities.Role;
+import org.springframework.beans.BeanUtils;
+import org.springframework.core.convert.converter.Converter;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.HashSet;
+import java.util.Set;
+
+public class EmployeeEntityToDTOConverter implements Converter<Employee, EmployeeDTO> {
+    @Override
+    public EmployeeDTO convert(Employee source) {
+        EmployeeDTO target = new EmployeeDTO();
+        BeanUtils.copyProperties(source, target);
+        target.setWorkExperience(ChronoUnit.YEARS.between(target.getEmploymentDate(), LocalDate.now()));
```

Comment:
workExperience is a strange field. If we have employment date and work experience is always calculated - maybe we should remove this field from the database completely?

Reply:
the field workExperience is not a DB field, it presents only in DTO

Reply:
ok